OrbitVulkanLayerClientGgp/LayerData.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "LayerData.h"
```

Comment:
LayerOptions.{h,cpp}  would be the better name imo. Feel free to ignore ...

Reply:
I think this rename makes sense and it was a quick change so I have renamed the class and variable, thanks for the suggestion!